common/src/main/java/org/candlepin/common/filter/LoggingFilter.java

```diff
@@ -95,11 +94,11 @@ public void doFilter(ServletRequest request, ServletResponse response,
 
             if (correlationId != null) {
                 if (CSID_REGEX.matcher(correlationId).matches()) {
-                    MDC.put(CSID_KEY, correlationId);
+                    MDC.put(CSID_KEY,
```

Comment:
I really don't feel truncating is the right approach here, but if that's what we're going to do, we should still output a log message indicating that we've truncated the original value down, and what that value was so we can at least track and figure out what happened in the event that it happens or when there's a collision caused by it.

Reply:
Fixed.